package.json

```
@@ -17,13 +17,16 @@
     "changeset": "changeset",
     "version": "changeset version && npm install",
     "publish": "npm run clean && npm run build -- --force && changeset publish && git push --follow-tags",
-    "publish:ci": "npm run build && changeset publish && git push --follow-tags"
+    "publish:ci": "npm run build && changeset publish && git push --follow-tags",
+    "prepare": "husky"
```

`prepare` is automatically run durning `npm install` - that means these precommit hooks will always run unless users explicitly opt out. This might be off-putting if someone is particular about their git workflow (or they already have linting from VSCode or the build is running in CI...). Can we just change this to a different command name, making it easy, but not automatic, to opt in? Maybe something like:

```suggestion
    "install-precommit-hooks": "husky"
```

---

Ah, yes, definitely did not mean to trigger it automatically!